gsa/src/gmp/parser.js

```diff
@@ -48,16 +46,9 @@ export const parseText = text => {
 
 export const parseTextElement = (text = {}) => {
   if (isDefined(text.__text)) {
-    return {
-      text: text.__text,
-      textExcerpt: text.__excerpt,
-    };
+    return text.__text;
   }
-
-  return {
-    text,
-    textExcerpt: '0',
-  };
+  return text;
```

Comment:
What's the intention of changing these lines?

Reply:
There was none. Probably this was accidentally ... i can't remember changing it ..

Reply:
Is it possible that husky/lint/whatever has done this?! I really did not touch these lines ...

Reply:
I don't think so. That's why I am always using `git add -p` ;-)